project/ws/app/src/lib/routes/home/components/request-list/request-list.component.ts

```diff
@@ -7,11 +7,13 @@ import { ActivatedRoute, Router } from '@angular/router'
 import { MatDialog, MatSnackBar, MatTableDataSource } from '@angular/material'
 import { ConfirmationBoxComponent } from '../../../training-plan/components/confirmation-box/confirmation.box.component'
 import { AssignListPopupComponent } from './assign-list-popup/assign-list-popup.component'
+import { LoaderService } from 'src/app/services/loader.service'
```

Comment:
Please use relative path, instead of actual path

Reply:
added relative path